### PR TITLE
Global mail config for address and name 

### PR DESCRIPTION
**Pull request by adrum**
This is a fix for #153

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -26,6 +26,8 @@ MAIL_PORT=2525
 MAIL_USERNAME=null
 MAIL_PASSWORD=null
 MAIL_ENCRYPTION=null
+MAIL_FROM_ADDRESS=regis@monicahq.com
```
Comment (djaiss) [inline .env.example]: Please replace this by `ValidEmailAddress` or something, so this email is never used by people ever.

File: .env.example [hunk 2]
```diff
@@ -26,6 +26,8 @@ MAIL_PORT=2525
...
+MAIL_FROM_NAME="Regis from Monica"
```
Comment (djaiss) [inline .env.example]: Please replace this by `YourName` or something.

File: .env.example (hunk 1 above)
Comment (djaiss) [inline .env.example]: The problem is that people will copy this example file and barely touch the default if they don't have to. By removing MY info entirely, we are sure this won't pop up in my email ever.

Comment (adrum) [inline .env.example]: Yup, that's a good call.

**Pull request merged**